Source/CombatExtended/CombatExtended/Loadouts/ITab_Inventory.cs

```diff
@@ -349,8 +349,36 @@ private void DrawThingRow(ref float y, float width, Thing thing, bool showDropBu
                             floatOptionList.Add(equipOption);
                         }
                     }
+                    //Thing we're hovering over is a shield
+                    if (thing is Apparel_Shield)

```

Comment:
No reason to check this then cast to Apparel below, we can use C#7 features :smile: 

`if (thing is Apparel_Shield shield)`

Reply:
I have no idea what that means XD If you suggest a code change that uses those I will implement/accept them